CHANGELOG.rst

```diff
@@ -2,6 +2,210 @@
+
+- Agent integration metadata payloads now include the JMX integrations.
+
+- Allow users to configure the HTTP timeout for the Logs Agent.
+
+- No longer have the Logs Agent fall back to TCP when configuring `logs_config.logs_dd_url` with a http(s):// prefix.
+
+- If the Oracle `can_connect` check is critical, also set the
+  `can_query` check to critical.
+
+- Display the number of times each log processor has been used in the Logs Agent status endpoint.
+
+- Reduce binary size by removing the Sensitive Data Scanner (SDS) from the logs agent.
+
+- OTLP spans support `db.namespace` semantic and map to `db.name` for DBM support.
+
+- Generate a more detailed warning when the Logs Agent tailer limit is reached.
+
+- Improved the granularity of the Logs Agent pipeline monitor to track the capacity of each individual component of the pipeline.
+
+- Remote Agent management operations on Windows now attempt to force stop the
+  Agent services if they do not respond to Service Control Manager requests.
+
+- Remote Agent management on Windows now automatically retries when the MSI
+  returns error 1618 (``ERROR_INSTALL_ALREADY_RUNNING``).
+
+
+.. _Release Notes_7.69.0_Bug Fixes:
+
+Bug Fixes
+---------
+
+- Correctly respect the `ecs_collect_resource_tags_ec2` variable when calling
+  the ECS Agent.  Start caching tags to reduce burden on the ECS Agent.
+  Start logging error responses from the ECS agent.
+
+- Fix a panic in Docker streams log parsing when stream messages are truncated on transmission.
+
+- Fix the cgroup reader bug that would prevent the generic container
+  check from sending metrics when the Agent encountered a permission error.
+
+- Fixes invalid logs compression error in DDOT, sets DDOT logs compression to gzip.
+
+- Add support for selecting the endpoint resolution method using
+  advanced AD identifiers in Kubernetes endpoint check configurations
+  defined in files or configmaps. This enables static pod check configurations
+  to correctly resolve the endpoint by setting resolve method to "ip".
+
+- Fixed the serializer exporter for the OSS Collector, which was not setting the correct proxy variables when sending metric data.
+
+- Fixed Windows installer overwriting ``install_info`` from setup scripts.
+  When using Fleet Automation setup scripts, the subsequent MSI installation
+  now skips writing ``install_info`` via a new ``SKIP_INSTALL_INFO`` flag,
+  preserving the original setup script installation method tracking.
+
+- Fix Jetson check to correctly parse the output of tegrastats for Orin boards.
+
+- Fix incorrect `container.memory.kernel` value when running with Kernel >= 5.19 and cgroupv2
+
+- Breaking change - Fixes the Oracle service name tag to be `service_name` instead of
+  `service`. This corrects the conflict with the APM `service` tag.
+  This is a breaking change for any users who had been relying on
+  the `service` tag to be set to the Oracle service name. The `service`
```

Comment:
```suggestion
- Breaking change - Fixes the Oracle service name tag to be ``service_name`` instead of
  ``service``. This corrects the conflict with the APM ``service`` tag.
  This is a breaking change for any users who had been relying on
  the ``service`` tag to be set to the Oracle service name. The ``service``
```